lib/pbench/server/cache_manager.py

```diff
@@ -1136,7 +1129,7 @@ def filestream(self, dataset_id: str, target: str) -> Optional[JSONOBJECT]:
             File info including a byte stream for a regular file
         """
         tarball = self.find_dataset(dataset_id)
-        return tarball.filestream(target)
+        return tarball.get_inventory(target)
```

Comment:
I can't explain how, but apparently we [have unit test coverage](http://jenkins.intlab.perf-infra.lab.eng.rdu2.redhat.com/job/pbench-ci-multi/job/PR-3473/4/cobertura/server/cache_manager_py/#1131) for line 1131 but not 1132.

Arguably, we don't need coverage for `CacheManager.get_inventory()` or `CacheManager.get_info()` since they are straight-line code with no conditionals, error handling, or transformations (although, it would be nice to have, easy to implement, and would keep us from getting distracted from gaps that we _should_ care about), but it's driving me nuts that I cannot find the unit test path which goes through line 1131...because I think it's indicating that we are mocking something improperly.